### PR TITLE
JSON Validator: Add Feature: Prettyprint JSON when it's valid

### LINKED ISSUE #3459: JSON Validator: prettyprint JSON when it's valid
## Background
The library used for the JSON Validator, [jsonlint](https://github.com/zaach/jsonlint), seems to also allow JSON prettyprinting when it's valid, in addition to outputting useful errors in case it isn't.

## To Do
- Broaden the trigger space (use triggers  like "json prettyprint", "json beautify", etc
- Edit the description to make it clear this IA doesn't just validate JSON, but beautifies it as well
- Either replace the input with its beautified version if it's valid, or show the beautified version next to it

## Forum
See the topic [Improve JSON Lint Goodie](https://forum.duckduckhack.com/t/improve-json-lint-goodie/327/4)

This issue is part of the [Programming Mission](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53): help us improve the results for [JavaScript related searches](https://forum.duckduckhack.com/t/javascript-search-overview/94)!

---------------------------------------------------
IA Page: https://duck.co/ia/view/json_validator
maintainer: @sahildua2305

### PR DESCRIPTION
**Pull request by sahildua2305**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [x] Bug fix: **`JSON Validator: Fix #3459`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Which issues (if any) does this fix?

Fixes #3459 

###### People to notify (@mention interested parties)
@MariagraziaAlastra 

---

Instant Answer Page: https://duck.co/ia/view/json_validator

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by edgesince84** [comment]
💪

File: share/goodie/jsonvalidator/triggers.txt [hunk 1]
```diff
@@ -6,3 +6,13 @@ validator json
...
+json formatter
+formatter json
+json prettyprint
+json pretty print
+prettyprint json
+pretty print json
+json prettify
+prettify json
+json lint
+json lint checker
```
Comment (MariagraziaAlastra) [inline share/goodie/jsonvalidator/triggers.txt]: @sahildua2305 What do you think about adding json beautifier, beautify json, visualize json, json visualizer, online json parser etc?

**Comment by MariagraziaAlastra** [comment]
Thanks @sahildua2305! It's on [beta](https://beta.duckduckgo.com/?q=json+validator&ia=answer) :smile:
I just left you a little comment about adding more triggers, otherwise looks good!

**Comment by MariagraziaAlastra** [comment]
@sahildua2305 Thanks, looks good! Merging 👍

**Comment by tagawa** [comment]
This is cool! We can point people to this when they're developing new cheat sheets. 😃

**Comment by sahildua2305** [comment]
Awesome @tagawa :smile: 

Let's flaunt it off on Twitter again :wink:

**Comment by edgesince84** [comment]
@tagawa is this something we could add to our docs?  I could have used it this morning!

**Comment by sahildua2305** [comment]
@edgesince84 I can go ahead and mention this in cheat sheet guide.

**Comment by tagawa** [comment]
@edgesince84 Definitely. We already recommend using https://jsonlint.com but this new IA should replace that. Perhaps we could mention it in more places in the docs if it's being missed.

@sahildua2305 - thank you, that would be great.

**Pull request merged**